GooglePlayInstant/Editor/AndroidBuildTools.cs

```diff
@@ -34,11 +34,19 @@ public static class AndroidBuildTools
             new Regex(@"^(\d+)\.(\d+)\.(\d+)(-rc(\d+))?$", RegexOptions.Compiled);
 
         /// <summary>
-        /// Returns the newest build-tools path as a string, or null if one couldn't be found.
+        /// Returns the build-tools path as a string.
```

Comment:
this is a really weird place for github to use as the anchor of this diff, I would have thought it would have used the next method